script/update_validator_avatars.rb

```diff
@@ -0,0 +1,19 @@
+#frozen_string_literal: true
+
+require "digest/md5"
+require "open-uri"
+
+TMP_STORAGE_PATH = Rails.root.join("storage", "validators", "avatars", "tmp").to_s.freeze
+
+NETWORKS.each do |network|
+  # next unless network == 'mainnet'
+
+  Validator.where(network: network).where.not(avatar_url: nil).each do |validator|
```

Comment:
Is there a point in processing inactive validators? I'd only process active ones.